Internal documentation [14071] (#161)
eProsima/DDS-Router: tools/ddsrouter_tool/README.md

# eProsima DDS Router Yaml Module

> This module create and executable that runs a DDS Router from a YAML configuration file.

Reviewer: ```suggestion
This module create an executable that runs a DDS Router configured via *yaml* configuration file.
```

---

> ## Example of use

Reviewer: ```suggestion
## Example of usage
```
